pkgs/applications/networking/cluster/cmctl/default.nix

```diff
@@ -25,6 +25,8 @@ buildGoModule rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
+  passthru.updateScript = ./update.sh;
```

Comment:
Sorry about the (likely annoying) minor feedback:

`passthru.updateScript` is usually added before meta. (To follow phases order)

Other than that, I suppose it will work. And I'm happy.

As [I'm no longer a commiter](https://github.com/NixOS/nixpkgs/issues/50105#issuecomment-1273053230), I won't be able to commit.

Reply:
No worries!

Apologies on the delay for opening this up.

Reply:
>passthru.updateScript is usually added before meta. (To follow phases order)

@superherointj Do you mean "just before meta"?